packages/js-sdk/test/Framework.test.js

```diff
@@ -99,13 +100,14 @@ contract("Framework class", accounts => {
             );
         }
 
-        it("without truffle framework", async () => {
-            const sf = new SuperfluidSDK.Framework({
-                web3Provider: web3.currentProvider
-            });
-            await sf.initialize();
-            testLoadedContracts(sf);
-        });
+        // it("without truffle framework", async () => {
```

Comment:
Intentionally commenting out this test, since using external web3 provider is going to be changed in #237